packages/datetime/src/dateInput.tsx

```diff
@@ -421,6 +425,7 @@ export class DateInput extends AbstractPureComponent2<DateInputProps, IDateInput
             this.handleDateChange(nextDate, true, true);
         } else if (e.which === Keys.TAB && this.state.isOpen) {
             this.getFirstTabbableElement()?.focus();
+            e.preventDefault();
```

Comment:
Necessary to prevent focusing the 2nd focusable element

Reply:
sounds like this should be a code comment

```suggestion
            // necessary to prevent focusing the 2nd focusable element
            e.preventDefault();
```

Reply:
I moved some things around and don't need to this anymore